pkgs/tools/system/ts/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
   '';
 
   src = fetchurl {
-    url = "http://viric.name/~viric/soft/ts/${name}.tar.gz";
+    url = "https://viric.name/~viric/soft/ts/ts-${version}.tar.gz";
```

Comment:
Seems like this could be simplified to

```
    url = "https://viric.name/~viric/soft/ts/${self.name}.tar.gz";
```

after https://github.com/NixOS/nixpkgs/pull/119942

(just noting. No action required)

Reply:
We shouldn't do this all over the place because there is no logical connection between the two values. They are just the same by coincidence and substituting parts of the URL just because we have some variable with the same content is annoying and unnecessary.

Reply:
That's a good point actually.

I've also just changed that PR for other reasons. You'd have to get it from `self.public.name` because the `${pname}-${version}` applies later anyway. This was not a valid use case.

`ts-${self.version}` still seems like a good one though.